.travis.yml

```diff
@@ -16,18 +16,16 @@ env:
   matrix:
     - PGVERSION=9.5
     - PGVERSION=9.6
+    - PGVERSION=10
 before_install:
   - git clone -b v0.6.2 --depth 1 https://github.com/citusdata/tools.git
   - sudo make -C tools install
   - setup_apt
-  - curl https://install.citusdata.com/community/deb.sh | sudo bash
   - nuke_pg
 install:
   - install_uncrustify
   - install_pg
   - install_custom_pg
-  # download and install HLL manually, as custom builds won't satisfy deps
-  - apt-get download "postgresql-${PGVERSION}-hll=2.10.1.citus-1" && sudo dpkg --force-confold --force-confdef --force-all -i *hll*.deb
```

Comment:
We haven't started building packages for HyperLogLog yet. I suppose we have two approaches:

  * Block this review on creating a HyperLogLog package for PostgreSQL 10
    * Quick approach: just build a one-off package for Ubuntu Trusty/PostgreSQL 10 and throw it in packagecloud to satisfy this line
    * Right approach: update `citus_package` to build for PostgreSQL 10
  * Leave this as-is (there are test variants that pass when HyperLogLog is absent) and add back when PostgreSQL 10 is released
  * Expand this line into a script that gracefully continues when HyperLogLog can't be installed (preserves existing behavior for 9.5/9.6 and punts on PostgreSQL 10)

Pick one and I'll do it.